conf/machine/include/package_revisions.inc

```diff
@@ -8,3 +8,6 @@ PV:pn-residentui = "4.5.7"
 PR:pn-residentui = "r0"
 SRCREV:pn-residentui = "3bfd5735a006b341d1a50825719fc04aec40eb0e"
 PACKAGE_ARCH:pn-residentui = "${APP_LAYER_ARCH}"
+
+PACKAGE_ARCH:pn-dab-adapter = "${APP_LAYER_ARCH}"
++PACKAGE_ARCH:pn-mosquitto = "${APP_LAYER_ARCH}"
```

Comment:
I cannot see SRCREV of mosquitto mentioned anywhere. Which version are we using ?
SRCREV of dab-adapter is present in its own recipe but RDKE has reference of that in package_revisions.inc of each layer. It would be better to add the same here as well.